.github/workflows/e2e_libvirt.yaml

```diff
@@ -180,12 +193,42 @@ jobs:
         env:
           GH_TOKEN: ${{ secrets.GITHUB_TOKEN }}
 
+      - name: Deploy CAA using Helm chart
+        if: ${{ inputs.install_method == 'helm' }}
+        env:
+          CAA_IMAGE: ${{ inputs.caa_image }}
+        run: |
+          kubectl create namespace confidential-containers-system || true
+
+          # Extract image name and tag from caa_image (format: registry/image:tag)
+          IMAGE_FULL="${CAA_IMAGE}"
+          IMAGE_TAG="${IMAGE_FULL##*:}"
+          IMAGE_NAME="${IMAGE_FULL%:*}"
+
+          echo "Deploying with Helm:"
+          echo "  Image: ${IMAGE_NAME}"
+          echo "  Tag: ${IMAGE_TAG}"
+
+          helm install peerpods ./install/charts/peerpods \
```

Comment:
Hi @beraldoleal , is this a temporary solution so we can test this PR before merge? or you want it as a long term solution? If the later then I prefer the installation being carried out inside the test tool.

Hmmm changes on the workflow won't take effect anyway because they run on pull_request_target: https://github.com/confidential-containers/cloud-api-adaptor/blob/main/.github/workflows/e2e_on_pull.yaml#L9

Reply:
ok. could you add a `TODO` so we don't loose track of that change?